packages/opc-feedback/src/opc-feedback.ts

```diff
@@ -18,25 +18,30 @@ import { repeat } from 'lit-html/directives/repeat.js';
 import { defaultTemplate } from './defaultTemplate';
 @customElement('opc-feedback')
 export class OpcFeedback extends LitElement {
-  @property({ type: String, attribute: 'url' }) url;
+  @property({ type: String, attribute: 'spaUrl' }) spaUrl;
+  @property({ type: String, attribute: 'docUrl' }) docUrl;
   @property({ reflect: true }) theme;
   @property({ type: Object }) template = defaultTemplate;
   @internalProperty()
   _openConfirmationModal = false;
   @internalProperty()
   _openFeedbackModal = false;
+  @internalProperty()
+  _openInitialModal = false;
+  @internalProperty()
+  _openBugModal = false;
   @internalProperty() protected _summary = '';
   @internalProperty() _experience = '';
   @internalProperty()
   _error = '';
-  _errorList = [];
 
   @query('textarea') textarea: HTMLTextAreaElement;
 
   constructor() {
     super();
     this.theme = "red";
-    this.url = "/feedback";
+    this.spaUrl = "/feedback";
+    this.docUrl = "/get-started";
```

Comment:
Can you please remove these default assignments from the constructor and add where you are declaring these properties